lib/ckeditor/dimagi-README.md

```diff
@@ -9,6 +9,7 @@ Use the following plugins:
 * floatingspace
 * undo (for change events)
 * widget
+* [config helper](http://ckeditor.com/addon/confighelper) (for placeholder support)
```

Comment:
It supports placeholders. like "Hint: drag a question here". So that when we define an input like `droppableText` whenever we initialize it there is still a placeholder